share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -378,6 +378,7 @@ A convenience function that can edit files in-place. For example:
 ```ruby
 inreplace "path" do |s|
   s.gsub! /foo/, "bar"
+  s.gsub! "123", "456"
```

Comment:
This code example is preceded by

>  You can also use the block form:

My suggestion would be to make it a bit more explicit:

>  You should use the block form if you need to make multiple replacements in a file: